particula/aerosol_dynamics/parcel_test.py

```diff
@@ -18,21 +18,88 @@
     charge=1,
 )
 
+standard_environment = environment.Environment(
+    temperature=298 * u.K,
+    pressure=101325 * u.Pa,
+)
+
 # create a parcel
-simple_parcel = parcel.Parcel('simple', temperature=300, pressure=101325)
+simple_parcel = parcel.Parcel('simple', standard_environment)
 
 # add single particle to parcel
 simple_parcel.add_particle(small_particle)
 simple_parcel.add_particle(large_particle)
-simple_parcel.create_and_add_particle('org1', 50e-9)
+simple_parcel.create_and_add_particle('org1', 500*u.nm)
 # add multiple particles to parcel
-simple_parcel.create_and_add_list_of_particle('org2', [1e-9, 2e-9, 3e-9])
+simple_parcel.create_and_add_list_of_particle('org2', [1e-9, 2e-9, 3e-9] * u.m)
+simple_parcel.create_and_add_list_of_particle(
+    'org2', [1e-9, 2e-9, 3e-9] * u.m,
+    [1.8, 1, 1] * u.kg / u.m ** 3, [1, 0, 2] * u.dimensionless
+)
 
 
 def test_getters():
     """
     Test that the getters work by confirming particle creation
     """
+    assert len(simple_parcel.particle_classes()) == 9
+    assert len(simple_parcel.particle_mass()) == 9
+    assert len(simple_parcel.particle_radius()) == 9
+    assert len(simple_parcel.particle_charge()) == 9
+
+
+def test_particle_mass_units():
+    '''
+    Test that the mass of the particles is returned in kg
+    '''
+    assert sum([i.mass().check('kg')
```

Comment:
Spread parentheses, then you can start a new line whenever you want without linters being unhappy.
e
```python
    assert sum(
          line
          line2
          half_line
    )
```

Reply:
like that?